etc/profile-m-z/telegram.profile

```diff
@@ -41,7 +44,7 @@ seccomp.block-secondary
 shell none
 
 disable-mnt
-private-bin telegram,Telegram,telegram-desktop
+private-bin bash,sh,telegram,Telegram,telegram-desktop
```

Comment:
private-bin bash,sh,telegram,Telegram,telegram-desktop,xdg-open

Reply:
> private-bin bash,sh,telegram,Telegram,telegram-desktop,xdg-open

Done. Thank you.